docs/src/components/home/GoldSponsors.tsx

```diff
@@ -61,7 +61,12 @@ const BACKLINKs = [
   {
     name: 'SocialWick',
     description: 'Buy Instagram followers.',
-    href: 'https://www.socialwick.com/instagram/followers',
+    href: 'https://www.socialwick.com/instagram/followers/?utm_source=mui.com&utm_medium=referral&utm_content=homepage',
```

Comment:
https://www.socialwick.com/instagram/followers/?utm_source=mui.com&utm_medium=referral&utm_content=homepage redirect to this URL, it looks like they don't want the UTM tags, which I think is fine, not everyone look at them in their analytics.

```suggestion
    href: 'https://www.socialwick.com/instagram/followers/',
```